File: C/1018-cedulas.c
#include <stdio.h>
 
int main() {
 
    int N,valor, cem , cinquenta , vinte , dez , cinco , dois , um;

    scanf("%d",&valor);
    
    N = valor;

    cem = N / 100;
    N = N - (cem *100);

    cinquenta = N / 50;
    N = N - (cinquenta * 50);

    vinte = N / 20;
    N = N - (vinte * 20);

    dez = N /10;
    N = N -(dez * 10);

    cinco = N / 5;
    N = N - (cinco * 5);

    dois = N / 2;
    N = N - (dois * 2);
    
    um = N;

    printf("%d\n",valor);
    printf("%d nota(s) de R$ 100,00\n",cem);
    printf("%d nota(s) de R$ 50,00\n",cinquenta);
    printf("%d nota(s) de R$ 20,00\n",vinte);
    printf("%d nota(s) de R$ 10,00\n",dez);
    printf("%d nota(s) de R$ 5,00\n",cinco);
    printf("%d nota(s) de R$ 2,00\n",dois);
    printf("%d nota(s) de R$ 1,00\n",um);
 
    return 0;
}

File: PYTHON/1018-cedulas.py
# -*- coding: utf-8 -*-

num = int(input())

valor = num

cem = num/100
aux = num%100

cinquenta = aux/50
aux = aux%50

vinte = aux/20
aux = aux%20

dez = aux/10
aux = aux%10

cinco = aux/5
aux = aux%5

dois = aux/2
aux = aux%2

um = aux/1

print(valor)
print("{} nota(s) de R$ 100,00".format(int(cem)))
print("{} nota(s) de R$ 50,00".format(int(cinquenta)))
print("{} nota(s) de R$ 20,00".format(int(vinte)))
print("{} nota(s) de R$ 10,00".format(int(dez)))
print("{} nota(s) de R$ 5,00".format(int(cinco)))
print("{} nota(s) de R$ 2,00".format(int(dois)))
print("{} nota(s) de R$ 1,00".format(int(um)))